tests/org.jboss.tools.cdi.bot.test/src/org/jboss/tools/cdi/bot/test/CDITestBase.java

```diff
@@ -164,6 +168,42 @@ protected void createClass(String project, String packageName, String className)
 		page.setName(className);
 		c.finish();
 	}
+	
+	protected void createInterface(String project, String packageName, String interfaceName) {
```

Comment:
@zcervink please try to refactor these three methods, too much duplicated code

Reply:
refactoring done, lets wait for PR check